Snakefile

```diff
@@ -76,7 +76,9 @@ bmt_converter = f'{bmt_marian_dir}/marian-conv'
 kenlm = f'{cwd}/3rd_party/kenlm'
```

Comment:
let's replace those to `{third_party_dir}/` too

Reply:
Done